processor/src/test/java/org/mapstruct/ap/test/nestedtargetproperties/NestedTargetPropertiesTest.java

```diff
@@ -51,11 +50,11 @@
 @RunWith(AnnotationProcessorTestRunner.class)
 public class NestedTargetPropertiesTest {
 
-    @Rule
-    public GeneratedSource generatedSource = new GeneratedSource().addComparisonToFixtureFor(
-        ChartEntryToArtist.class,
-        ChartEntryToArtistUpdate.class
-    );
+//    @Rule
+//    public GeneratedSource generatedSource = new GeneratedSource().addComparisonToFixtureFor(
+//        ChartEntryToArtist.class,
+//        ChartEntryToArtistUpdate.class
+//    );
```

Comment:
@navpil now that I've read your comment above a bit more careful: if you can make sure that the order remains "stable" in a sense that it's not depending on any HashMap/HashSet iteration, then it's totally okay to adapt the reference impl file.
But if this is a result of HashMap/HashSet iteration (which can vary based on the JDK version, among other things), then we should somehow fix it.